corehq/motech/repeaters/tests/test_repeater.py

```diff
@@ -1284,6 +1287,65 @@ def test_five_retries(self):
             self.assertEqual(interval, timedelta(hours=expected_interval_hours))
 
 
+class DataSourceRepeaterTest(BaseRepeaterTest, TestXmlMixin):
```

Comment:
why this inheriting `TextXmlMixin`?

Reply:
2c7d316dd79acca4967526f08bedd20ca0887451